mindjunkies/courses/forms.py

```
@@ -57,5 +57,18 @@ def save(self, commit=True):
 
+
+class CourseTokenForm(forms.ModelForm):
+    class Meta:
+        model = CourseToken
+        fields = ['course']  # Only allow selecting a course; status & user are handled automatically.
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.fields['course'].empty_label = "Select a course"
+        self.fields['course'].widget.attrs.update({
+            'class': 'block w-full p-2 border border-gray-300 rounded-md shadow-sm focus:ring focus:ring-blue-500 focus:border-blue-500',
```

Ruff (0.8.2)

Line too long (137 > 119)

(E501)